applications/serial_lte_modem/doc/TWI_AT_commands.rst

```diff
@@ -0,0 +1,227 @@
+.. _SLM_AT_TWI:
+
+TWI AT commands
+***************
+
+.. contents::
+   :local:
+   :depth: 2
+
+The following commands list contains TWI related AT commands.
+
+Open TWI instance #XTWIOP
+=========================
+
+The ``#XTWIOP`` command opens TWI instance.
+
+Set command
+-----------
+
+The set command allows you to open TWI instance.
+
+Syntax
+~~~~~~
+
+::
+
+   #XTWIOP=<index>
+
+The ``<index>`` parameter accepts the following integer values:
+
+* ``0`` - i2c0
+* ``1`` - i2c1
+* ``2`` - i2c2
+* ``3`` - i2c3
+
+Example
+~~~~~~~
+
+::
+
+   AT#XTWIOP=2
+   OK
+
+Read command
+------------
+
+The read command is not supported.
+
+Test command
+------------
+
+The test command is not supported.
+
```

Comment:
Do you think it'll be good to support Test to list all available TWI instance by DTS?

Reply:
#XTWILS is added to list all available TWI instance.